_new_content_templates/blank_releasenotes.adoc

```diff
@@ -25,19 +25,19 @@ The following "list" should be added to `data/nav.yml`
 
 == Overview
 
-{productname} vnumtxt was released for {enterpriseversion} and {cloudname} on <<`WEEKDAY`, `MMM` `D`^`st|nd|th`^, `YYYY`>>. It includes {productname} vnumtxt and additional changes to premium plugins. These release notes provide an overview of the changes for {productname} vnumtxt, including:
-
-* <<newfeatures,New features>>
-* <<enhancements,Enhancements>>
-* <<functionalitychanges,Functionality changes>>
-* <<accompanyingpremiumpluginchanges,Accompanying Premium Plugin changes>>
-* <<accompanyingpremiumskinsandiconpackschanges,Accompanying Premium Skins and Icon Packs changes>>
-* <<accompanyingpremiumself-hostedserver-sidecomponentchanges,Accompanying Premium self-hosted server-side component changes>>
-* <<generalbugfixes,General bug fixes>>
-* <<securityfixes,Security fixes>>
-* <<deprecatedfeatures,Deprecated features>>
-* <<knownissues,Known issues>>
-* <<upgradingtothelatestversionoftinymce5,Upgrading to the latest version of TinyMCE 5>>
+{productname} vnumtxt was released for {enterpriseversion} and {cloudname} on +`WEEKDAY`, `MMM` `D`^`st|nd|th`^, `YYYY`+ It includes {productname} vnumtxt and additional changes to premium plugins. These release notes provide an overview of the changes for {productname} vnumtxt, including:
+
+* xref:newfeatures[New features]
+* xref:enhancements[Enhancements]
+* xref:functionalitychanges[Functionality changes]
+* xref:accompanying-premium-self-hosted-server-side-component-changes[Accompanying Premium Plugin changes]
+* xref:accompanyingpremiumskinsandiconpackschanges[Accompanying Premium Skins and Icon Packs changes]
+* xref:accompanying-premium-self-hosted-server-side-component-changes[Accompanying Premium self-hosted server-side component changes]
+* xref:general-bug-fixes[General bug fixes]
+* xref:security-fixes[Security fixes]
+* xref:deprecatedfeatures[Deprecated features]
+* xref:knownissues[Known issues]
+* xref:upgrading-to-the-latest-version-of-tinymce-5[Upgrading to the latest version of TinyMCE 5]
```

Comment:
```suggestion
* xref:new-features[New features]
* xref:enhancements[Enhancements]
* xref:functionality-changes[Functionality changes]
* xref:accompanying-premium-self-hosted-server-side-component-changes[Accompanying Premium Plugin changes]
* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
* xref:accompanying-premium-self-hosted-server-side-component-changes[Accompanying Premium self-hosted server-side component changes]
* xref:general-bug-fixes[General bug fixes]
* xref:security-fixes[Security fixes]
* xref:deprecated-features[Deprecated features]
* xref:known-issues[Known issues]
* xref:upgrading-to-the-latest-version-of-tinymce-5[Upgrading to the latest version of TinyMCE 5]
```

Reply:
Not going to worry about this, there shouldn't be any new release notes. I didn't use this template anyway when migrating the live release notes.